libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/utils/BitmapEncodeOptions.java

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.navigation.ui.internal.utils;
+
+import com.google.auto.value.AutoValue;
+
+import static com.mapbox.navigation.ui.internal.utils.ViewUtils.DEFAULT_BITMAP_ENCODE_COMPRESS_QUALITY;
+import static com.mapbox.navigation.ui.internal.utils.ViewUtils.DEFAULT_BITMAP_ENCODE_WIDTH;
+
+@AutoValue
```

Comment:
Can we use a vanilla, Kotlin builder pattern? We'll not rely on autovalue when the codebase is fully migrated to Kotlin.